docs/source/developer/release.md

```diff
@@ -7,12 +7,18 @@
 
 1. Create a new branch and open a PR on GitHub which will run all actions
 
-1. Apply any fixes to make the actions pass 
+1. Apply any fixes to make the actions pass
+
+1. Clean your local directory. This will prevent any files not under source control from being built into the distribution.
```

Comment:
maybe a friendly note for people skimming and who don't realize the power that they possess:

*__note__: this cannot be undone. please take care to move any important files elsewhere. to review files that will be removed, run `git status`*.

Reply:
yeah this is a good suggestion (sorry for jumping the gun on the merge!)

Reply:
oh, the PR was still open when i wrote the comment, i must have had a stale web page. np duder!